SpatialGDK/Source/SpatialGDKTests/SpatialGDKServices/LocalDeploymentManager/LocalDeploymentManagerUtilities.cpp

```diff
@@ -101,6 +101,12 @@ bool FStopDeployment::Update()
 {
 	FLocalDeploymentManager* LocalDeploymentManager = GetLocalDeploymentManager();
 
+	if (LocalDeploymentManager->IsDeploymentStarting())
+	{
+		// Wait for deployment to finish starting before stopping it
+		return false;
+	}
```

Comment:
Does this retry the stop once the starting finishes? Otherwise, can we log a warning indicating the deployment is still starting & users should retry later?

Reply:
This method is used for automation testing
In this method, if `true` is returned, then the test moves onto the next automation step
On the other hand, if `false` is returned, then this step is retried 

So, to answer your question, the stop is retried after starting finishes 🙂